packages/manager/modules/account-sidebar/src/user-infos/template.html

```diff
@@ -25,9 +25,15 @@
             ></span>
         </p>
         <p
+            data-ng-if="$ctrl.role !== 'provider'"
```

Comment:
Better as constant (one single place to make changes).
```suggestion
            data-ng-if="$ctrl.role !== $ctrl.USER_ROLE_PROVIDER"
```